lib/resources/r4/document_reference.yaml

```
@@ -290,3 +291,42 @@ fields:
         "end": "2020-01-01T01:00:00.000Z"
       }
     }
+
+- name: docStatus
+  required: 'Yes'
+  type: code
+  action:
+    - update
+  description: The status of the underlying document.
+  note: Only `final` is supported for provider access. `final` and `amended` are supported for system access.
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.docStatus
+  example: |
+    {
+      "docStatus": “amended”
+    }
+
+- name: type
+  required: ‘Yes’
+  type: CodeableConcept
+  action:
+  - update
+  description: Specifies the particular kind of document referenced.
+  note: The type must include a LOINC or a proprietary coding but not both together. Multiple LOINC codings or a single proprietary coding can be provided.
+        <br/><br/>
+        When providing proprietary code system, it should be of format 'https://fhir.cerner.com/&lt;your EHR source id&gt;/codeSet/&lt;code set&gt;' (where code set is a Millennium code set). Example&#58; 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/72'.
```

We're only allowing proprietary codes from codeset 72.  I think we can make that a little more clear here.

---

For consistency, I thought I'd make it like create's type. Should we change both?

---

Yup, if it makes it seem like we could support multiple codeSets, we should.... a simple change might be to just do this?
```suggestion
        When providing proprietary code system, it should be of format 'https://fhir.cerner.com/&lt;your EHR source id&gt;/codeSet/&lt;code set&gt;' (where code set is a Millennium code set 72). Example&#58; 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/72'.
```

---

You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/463/commits/2697a167ece46ac0a9e9b0a2a671208d4ce9715d